tests/golden/check-ignoreExternalRefsTo/check-ignoreExternalRefsTo.bats

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env bats
+ #!/usr/bin/env bats
```

Comment:
Was this change made on purpose?

Reply:
No, thanks)

Reply:
This is my last nitpick ^^ I'll approve once this is reverted

Reply:
Ok, let's merge this. Fixed and rebased

Reply:
As I understand, we shouldn't add anything to changelog (while those changes are visible to user)

Reply:
Yup that's right